elyra/pipeline/processor.py

```diff
@@ -243,6 +256,16 @@ def _sort_operation_dependencies(operations_by_id: dict, ordered_operations: lis
 
 
 class RuntimePipelineProcess(PipelineProcessor):
+    component_registry: ComponentRegistry = ComponentRegistry()
+
+    def get_components(self):
+        # component_registry = ComponentReader.get_instance(registry_type='file', processor_type=self.type)
```

Comment:
I'm confused.  What is the difference between this method and the same method and identical implementation on `PipelineProcessorManager`?

Reply:
Looks like this just a holdover. I'll remove the one in `RuntimePipelineProcess` in favor of the one in `PipelineProcessor`  since the former inherits from the latter.